src/author.py

```diff
@@ -10,13 +10,13 @@ def __init__(self, name, email):
         self.__time = str(dt.today())
 
     def set_name(self, name) -> None:
-        if eh_nulo_ou_vazio(name):
+        if is_it_null_ou_empty(name):
```

Comment:
haha, o it gera concordância mas normalmente não é usado, é comum ver IsNullOrEmpty por exemplo (o case depende da linguagem), mas ficou o ou ali em vez de or né?